docs/_guides/codewind-eclipse-quick-guide.md

```diff
@@ -37,61 +37,61 @@ The Codewind installation pulls the following images that form the Codewind back
+1. The Eclipse plug-in installs when you install Codewind from the [Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind). Or from the Eclipse IDE, you can go to **Help** > **Eclipse Marketplace** then search for Codewind. 
 2. The Codewind backend containers install after you click `Install`. Clicking `Install` downloads the Codewind backend containers, ~1GB.
 
 ### Configuring Codewind to use application stacks
 Configure Codewind to use Appsody templates so you can focus exclusively on your code. These templates include an Eclipse MicroProfile stack that you can use to follow this guide. Complete the following steps to select the Appsody templates:
 
-1. Click the `Codewind` tab.
-2. Expand `Codewind` by clicking the drop-down arrow.
-3. Right-click `Local [Running]`.
-4. Select `Manage Template Sources...`.
-5. Select `Appsody Stacks - incubator`.
-6. Click the `OK`.
+1. Click the **Codewind** tab.
+2. Expand **Codewind** by clicking the drop-down arrow.
+3. Right-click **Local [Running]**.
+4. Select **Manage Template Sources...**.
+5. Select **Appsody Stacks - incubator**.
+6. Click **OK**.
 
 When you configure Codewind to use Appsody templates, continue to develop your microservice within Codewind.
 
 If your organization uses customized application stacks and gives you a URL that points to an `index.json` file, you can add it to Codewind:
 
-1. Return to  `Codewind` and right-click `Local [Running]`.
-2. Select `Manage Template Sources...`.
-3. Click `Add...` to add your URL.
+1. Return to  **Codewind** and right-click **Local [Running]**.
+2. Select **Manage Template Sources...**.
+3. Click **Add...** to add your URL.
 4. Add your URL in the `URL:` box in the pop-up window and save your changes.
 
 ### Creating an Appsody project
 Appsody helps you develop containerized applications and removes the burden of managing the full software development stack. If you want more context about Appsody, see the [Appsody welcome page](https://appsody.dev/docs).
 
-1. Right-click `Local [Running]` under `Codewind` in the `Codewind` tab.
-2. Select `+ Create New Project...`
+1. Right-click **Local [Running]** under **Codewind** in the **Codewind** tab.
+2. Select **+ Create New Project...**
     * **Note:** Make sure that Docker is running. Otherwise, you get an error.
-3. Name your project `appsody-calculator`.
-4. Under `Template`, select `Appsody Open Liberty default template`. 
-    * If you don't see an Appsody template, select the `Manage Template Sources...` link at the end of the window.
-    * Select the `Appsody Stacks - incubator` checkbox.
-    * Click `OK`.
+3. Name your project **appsody-calculator**.
+4. Under **Template**, select **Appsody Open Liberty default template**. 
+    * If you don't see an Appsody template, select the **Manage Template Sources...** link at the end of the window.
```

Comment:
For accessibility reasons, I don't think we should mention "at the end of the window." If the window has a name, though, we might say, "...in the **name** window."